requirements/scenarios.md

```diff
@@ -26,26 +26,26 @@ However, they do wish to add attestations to the validity of the content.
 1. The Wabbit Networks company builds their `net-monitor` software.
       - As a result of the build, they produce an [OCI Image][oci-image], a Software Bill of Materials (`SBoM`) and to comply with gpl licensing, produce another artifact which contains the source (`src`) to all the gpl licensed projects.
       - The `SBoM` and `src` artifacts are created as reference types to the image, creating a graph of artifacts.
-      - Each of the artifacts are signed with the Notary v2 wabbit-networks key.
-1. The Wabbit Networks signed contents are pushed to a public OCI compliant registry.
+      - Each of the artifacts are signed with the wabbit-networks key.
+2. The Wabbit Networks signed contents are pushed to a public OCI compliant registry.
```

Comment:
Looks like your markdown linter updated the bullets, making it harder for future edits. 
Can you revert the incremental numbering, for this line and those below, to all be `1.` and let markdown renders resolve the ordering?

Reply:
I think I need to disable the linter, since it updated the numbering and TOC automatically.